cmake/MMDeploy.cmake

```diff
@@ -12,6 +12,18 @@ function (mmdeploy_export NAME)
             RUNTIME DESTINATION bin)
 endfunction ()
 
+macro(mmdeploy_add_net NAME)
+    if (MMDEPLOY_DYNAMIC_BACKEND)
+        # remove unused
+        cmake_parse_arguments(_NET "EXCLUDE;STATIC;SHARED;MODULE" "" "" ${ARGN})
+        mmdeploy_add_library(${NAME} SHARED EXCLUDE ${_NET_UNPARSED_ARGUMENTS})
+        target_link_libraries(${PROJECT_NAME} PRIVATE mmdeploy)
```

Comment:
`mmdeploy` or `mmdeploy_core`?

Reply:
mmdeploy, we should link net.so to the dynamic library.